rentlinx.gemspec

```diff
@@ -12,4 +12,5 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'httpclient', '~> 2.6.0'
   s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'phony'
```

Comment:
Fair point. Maybe you could just specific the major version dependency then and hope a minor version doesn't break the gem. In general APM shouldn't bump a gem version if it has the potential to break other gems.